package.json

```diff
@@ -8,9 +8,164 @@
+    "generate:metadata:graph": "npx ts-node -r tsconfig-paths/register apps/graph-api/src/generate-metadata.ts",
+    "generate:metadata": "npm run generate:metadata:account ; npm run generate:metadata:content-publishing ; npm run generate:metadata:content-watcher ; npm run generate:metadata:graph",
+    "generate:openapi:account": "nest start account --entryFile build-openapi",
+    "generate:openapi:content-publishing": "nest start content-publishing-api --entryFile build-openapi",
+    "generate:openapi:content-watcher": "nest start content-watcher --entryFile build-openapi",
+    "generate:openapi:graph": "nest start graph-api --entryFile build-openapi",
+    "generate:openapi": "npm run generate:openapi:account ; npm run generate:openapi:content-publishing ; npm run generate:openapi:content-watcher ; npm run generate:openapi:graph",
+    "generate:types:graph": "npx openapi-client-axios-typegen openapi-specs/graph-webhooks.openapi.yaml > libs/graph-lib/src/types/webhook-types.d.ts ; prettier --write libs/graph-lib/src/types/webhook-types.d.ts",
+    "generate:types:content-watcher": "npx @hey-api/openapi-ts -i openapi-specs/content-announcement.openapi.json -o libs/content-watcher-lib/src/types/content-announcement ; prettier --write libs/content-watcher-lib/src/types/content-announcement",
+    "generate:types": "npm run generate:types:graph ; npm run generate:types:content-watcher",
+    "generate:swagger-ui:account": "npx --yes @redocly/cli build-docs openapi-specs/account.openapi.json --output=./docs/account/index.html",
+    "generate:swagger-ui:content-publishing": "npx --yes @redocly/cli build-docs openapi-specs/content-publishing.openapi.json --output=./docs/content-publishing/index.html",
+    "generate:swagger-ui:content-watcher": "npx --yes @redocly/cli build-docs openapi-specs/content-watcher.openapi.json --output=./docs/content-watcher/index.html",
+    "generate:swagger-ui:graph": "npx --yes @redocly/cli build-docs openapi-specs/graph.openapi.json --output=./docs/graph/index.html",
+    "generate:swagger-ui": "npm run generate:swagger-ui:account ; npm run generate:swagger-ui:content-publishing ; npm run generate:swagger-ui:content-watcher ; npm run generate:swagger-ui:graph",
+    "test:account": "dotenvx run -f env-files/account.template.env -- jest 'account*'",
+    "test:content-publishing": "dotenvx run -f env-files/content-publishing.template.env -- jest 'content-publishing*'",
+    "test:content-watcher": "dotenvx run -f env-files/content-watcher.template.env -- jest 'content-watcher*'",
+    "test:graph": "dotenvx run -f env-files/graph.template.env -- jest 'graph*'",
+    "test": "npm run test:account ; npm run test:content-publishing ; npm run test:content-watcher ; npm run test:graph",
+    "test:verbose": "jest --coverage --verbose",
+    "test:e2e:account": "dotenvx run -f .env.account -- jest --silent --runInBand --testRegex \".e2e-spec.ts\" --detectOpenHandles --forceExit 'account*'",
```

Comment:
why only this one has `--silent --runInBand`?

Reply:
Just copied from the individual microservices. We have a backlog item to get e2e tests working in the pipeline; we'll probably clean this up then.